vmdb/app/views/layouts/listnav/_container_service.html.haml

```diff
@@ -30,3 +30,9 @@
               = link_to(_("Container Groups (%s)") % num_groups,
                   {:action  => 'show', :id => @record, :display => 'container_groups'},
                   :title => _("Show Container Groups"))
+        - if role_allows(:feature => "container_project_show") && !@record.container_project.nil?
+          %li
+            = link_to("#{ui_lookup(:table => "container_project")}: #{@record.container_project.name}",
+                {:controller => "container_project", :action => 'show', :id => @record.container_project.id.to_s},
+                {:title => _("Show this container service's parent %s") % ui_lookup(:table => "container_project")},
+                '/images/icons/16/link_external.gif')
```

Comment:
@screeley44 this last argument `'/images/icons/16/link_external.gif'` seems unneeded. Isn't it?